---
# ✈ Loading Data (Practice)
---
- Anas Abu Alhaija

For this assignment, you will be loading and previewing a dataset related to insurance costs.

[Insurance Data](https://drive.google.com/file/d/1-7mcBguuzAV7JWVu2XCpgAO-fO6X2cl0/view?usp=sharing)

In [10]:
# save a file path
fpath = "/content/drive/MyDrive/CodingDojo/01-Fundamentals/Week02/Data/insurance_mod.csv"

In [11]:
# import pandas
import pandas as pd

In [12]:
# load in data as a pandas dataframe
df_insurance = pd.read_csv(fpath)

In [13]:
# preview the first five rows
df_insurance.head()

age     sex     bmi  children  smoker     region  charges
0   19  female  27.900         0       1  southwest  16885.0
1   18    male  33.770         1       0  southeast   1726.0
2   28    male  33.000         3       0  southeast   4449.0
3   33    male  22.705         0       0  northwest  21984.0
4   32    male  28.880         0       0  northwest   3867.0

In [14]:
# Preview the last five rows
df_insurance.tail()

age     sex    bmi  children  smoker     region  charges
1333   50    male  30.97         3       0  northwest  10601.0
1334   18  female  31.92         0       0  northeast   2206.0
1335   18  female  36.85         0       0  southeast   1630.0
1336   21  female  25.80         0       0  southwest   2008.0
1337   61  female  29.07         0       1  northwest  29141.0

In [15]:
# obtain a summary table for each column
df_insurance.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1338 entries, 0 to 1337
Data columns (total 7 columns):
 #   Column    Non-Null Count  Dtype  
---  ------    --------------  -----  
 0   age       1338 non-null   int64  
 1   sex       1338 non-null   object 
 2   bmi       1338 non-null   float64
 3   children  1338 non-null   int64  
 4   smoker    1338 non-null   int64  
 5   region    1338 non-null   object 
 6   charges   1338 non-null   float64
dtypes: float64(2), int64(3), object(2)
memory usage: 73.3+ KB


Use the output of .info() to answer the following in a text cell:

a) How many rows are there❓

*  1338



b) How many columns are there❓

*  7


🔚